content/webapp/components/styled/AccessResources.ts

```diff
@@ -0,0 +1,55 @@
+import styled from 'styled-components';
+import PlainList from '@weco/common/views/components/styled/PlainList';
+import Space from '@weco/common/views/components/styled/Space';
+import { PaletteColor } from '@weco/common/views/themes/config';
+
+export const ResourcesList = styled(PlainList)`
+  display: flex;
+  flex-wrap: wrap;
+  gap: 15px;
```

Comment:
`gap` can be added with the `Space` component now – is that possible here?

Reply:
I can't combine PlainList and Space can I?